frontend/app/helpers/spree/frontend_helper.rb

```diff
@@ -364,5 +364,11 @@ def checkout_edit_link(step = 'address', order = @order)
         inline_svg_tag 'edit.svg'
       end
     end
+
+    def products_for_filtering
```

Comment:
@mad-eel let's move this to https://github.com/spree/spree/blob/master/core/app/models/concerns/spree/product_scopes.rb